visualization/app/codeCharta/ui/mapTreeView/mapTreeView.component.spec.ts

```diff
@@ -39,11 +52,81 @@ describe("MapTreeViewController", () => {
 
 			expect(CodeMapPreRenderService.subscribe).toHaveBeenCalledWith($rootScope, mapTreeViewController)
 		})
+
+		it("should subscribe to SortingDialogOptionService", () => {
+			SortingDialogOptionService.subscribe = jest.fn()
+
+			rebuildController()
+
+			expect(SortingDialogOptionService.subscribe).toHaveBeenCalledWith($rootScope, mapTreeViewController)
+		})
+		it("should subscribe to SortingOrderAscendingService", () => {
+			SortingOrderAscendingService.subscribe = jest.fn()
+
+			rebuildController()
+
+			expect(SortingOrderAscendingService.subscribe).toHaveBeenCalledWith($rootScope, mapTreeViewController)
+		})
+	})
+
+	describe("applySort", () => {
+		it("should return the children array sorted by unary", () => {
```

Comment:
also here in the test suite: use const, use empty lines between arrange, act, assert blocks